JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt

```diff
@@ -117,19 +116,17 @@ fun PostCardHistory(post: Post, navigateTo: (Screen) -> Unit) {
             modifier = Modifier.padding(end = 16.dp)
         )
         Column(Modifier.weight(1f)) {
-            ProvideEmphasis(AmbientEmphasisLevels.current.medium) {
```

Comment:
this medium emphasis was removed on purpose?

Reply:
I should create a bug for that. The migration tool on `AmbientEmphasisLevels` set the value automatically to `ContentAlpha.high` and I removed it after cleaning up the project